VelocitySDK/Sensors/Recorders/VLTSensorRecorder.m

```diff
@@ -93,12 +93,16 @@ - (void)addSample:(nonnull id<VLTSample>)sample
     id<VLTSample> lastObject        = [samples lastObject];
     if (lastObject) {
         NSTimeInterval timestampLimit = lastObject.timestamp - interval;
+        NSInteger lastValidSampleIndex = NSNotFound;
         for (NSInteger i = (samples.count - 1); i >= 0; i--) {
```

Comment:
I think, it would be nice to improve our code and do not iterate when that is not needed.